tools/ci.yml

```diff
@@ -32,6 +26,21 @@ services:
       - nginx
       - objectstore_init
 
+  currenttest:
```

Comment:
what are current tests?

Reply:
Oh, pushing that was my mistake. Just a container that only runs the test I was interested in